### PR TITLE
fix: setting calendardatepicker minwidth

### PR DESCRIPTION
**Pull request by rajamatt**
﻿GitHub Issue: closes https://github.com/unoplatform/uno/issues/11062

## PR Type

What kind of change does this PR introduce?

- Bugfix

## Description

- Added MinWidth attribute to the Grid that displays the picker containing the date.

image
image

## PR Checklist 
Please check if your PR fulfills the following requirements:

- Commits must be following the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)
- [x] Tested UWP
- [x] Tested iOS
- [x] Tested Android
- [x] Tested WASM
- [ ] Tested MacOS
- [x] Contains **No** breaking changes
  > If the pull request contains breaking changes, commit message must contain a detailed description of the action to take for the consumer of this library. As explained by the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)

### CODE REVIEW

File: src/library/Uno.Material/Styles/Controls/v2/CalendarDatePicker.xaml
Comment (kazo0) [inline src/library/Uno.Material/Styles/Controls/v2/CalendarDatePicker.xaml:1]: What does the Fluent style CalendarDatePicker look like if you were to remove the MaterialDatePickerStyle setter in your sample? Does it also force a MinWidth in the template?

Comment (rajamatt) [reply src/library/Uno.Material/Styles/Controls/v2/CalendarDatePicker.xaml:1]: When I inspect the xaml element, the Fluent CalendarDatePicker always stays at 122 width with placeholder text (even if I make the screen super tiny to where it cuts half of the picker) 
image
and when it has a selected date it always stays at 108 width.
image

Comment (rajamatt) [reply src/library/Uno.Material/Styles/Controls/v2/CalendarDatePicker.xaml:1]: @kazo0

File: src/library/Uno.Material/Styles/Controls/v2/CalendarDatePicker.xaml [hunk 1]
```diff
@@ -279,6 +279,7 @@
                                    Height="2" />
 
                         <Grid Margin="{ThemeResource CalendarDatePickerContentMargin}"
+                              MinWidth="76"
```
Comment (Xiaoy312) [inline src/library/Uno.Material/Styles/Controls/v2/CalendarDatePicker.xaml:282]: i think the problem might stems from the calendar having a fixed height
and that the TextBlock.TextWrap is set to wrap

can you change it to NoWrap and see without the min-width, the control would adjust itself?

**Review by Xiaoy312 — COMMENTED** [review]
looks good

File: src/library/Uno.Material/Styles/Controls/v2/CalendarDatePicker.xaml (hunk 1 above)
Comment (rajamatt) [reply src/library/Uno.Material/Styles/Controls/v2/CalendarDatePicker.xaml:282]: I just retested this issue and it is fixed on latest master, the date on Skia is no longer in YYYY-MM-DD format, it is now in MM-DD-YYYY format like on all the other platforms.

I'm not sure what fixed it, update of uno? Lightweight styling? I can close this PR anyway as it isn't needed anymore

**Pull request closed without merging**